### PR TITLE
Add doc about Performance and SEO

### PR DESCRIPTION
**Pull request by obliviusm**
Add doc with resources for checking Performance and SEO

### CODE REVIEW

**Comment by obliviusm** [comment]
@andriyParashchuk pls check

**Comment by obliviusm** [comment]
Deployed to https://jetthoughts-pr-367.surge.sh

File: .gitignore [hunk 1]
```diff
@@ -5,4 +5,5 @@ _site
 .jekyll-cache/
 package-lock.json
 node_modules/
+.DS_Store
```
Comment (andriyParashchuk) [inline .gitignore:8]: 👍

**Review by andriyParashchuk — APPROVED** [review]
LGTM

**Pull request merged**